src/parser/jobs/sam/modules/OGCDDowntime.ts

```diff
@@ -4,6 +4,8 @@ import {CooldownDowntime} from 'parser/core/modules/CooldownDowntime'
 // GCD timing: 2.5 seconds for first two GCDs (to get Shifu up), 2.18 seconds for all others.
 // meikyo is used after the 7th gcd
 const FIRSTUSEOFFSET_MEIKYO = 15900
+// Senei/Guren is used after 6th gcd
+const FIRSTUSEOFFSET_50KENKI = 13720
```

Comment:
How consistent is this, along with the first Meikyo offset?  Lag happens and there may somehow be players with a 2.5s GCD for whatever reason, plus regional differences on openers. It might be fine but I'd probably take a look at some ~90th percentile SAM logs across NA/EU/JP and see what players that play well but aren't necessarily perfect are doing, particularly in JP where they usually don't follow English guides.